components/server/src/prebuilds/github-enterprise-app.ts

```diff
@@ -174,10 +172,7 @@ export class GitHubEnterpriseApp {
                         context,
                     });
 
-                    const shouldRun = Project.hasPrebuildSettings(project)
-                        ? prebuildPrecondition.shouldRun
-                        : this.appRules.shouldRunPrebuild(config, CommitContext.isDefaultBranch(context), false, false);
```

Comment:
FYI, this does effectively disables comparable settings in .gitpod.yml, which we want to deprecate in https://github.com/gitpod-io/gitpod/pull/18788